src/main/java/io/appium/java_client/chromium/options/ChromiumOptions.java

```diff
@@ -0,0 +1,57 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * See the NOTICE file distributed with this work for additional
+ * information regarding copyright ownership.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.appium.java_client.chromium.options;
+
+import io.appium.java_client.remote.AutomationName;
+import io.appium.java_client.remote.options.*;
+import org.openqa.selenium.Capabilities;
+
+import java.util.Map;
+
+/**
+ * <a href='https://github.com/appium/appium-chromium-driver#usage'>appium-chromium-driver usage section</a>
+ */
+public class ChromiumOptions extends BaseOptions<ChromiumOptions> implements
```

Comment:
Do you want to add several simple unit tests to verify the functionality of chromium options?

Reply:
I was in doubt about writing unit tests. I usually do, even code in TDD mode. I saw the library's lack of unit tests, so I didn't create them. I'll add unit tests.

Reply:
the coverage is not ideal, but this is anyway a good practice to perform at least basic validation of newly added entries. Check https://github.com/appium/java-client/blob/master/src/test/java/io/appium/java_client/drivers/options/OptionsBuildingTest.java for example